### PR TITLE
Adding Context Path Convention to returned URI

### PR DESCRIPTION
**Pull request by cschroed-usgs**
Also moved from tags to version in ServiceConfig.

### CODE REVIEW

File: src/test/java/gov/usgs/cida/microservices/catalog/ConsulRegistrationAndDiscoveryClientsTestIT.java [hunk 1]
```diff
@@ -62,13 +63,13 @@ public static void setUpClass() throws IOException {
...
 				.setPort(port)
 				.setAddress(address)
-				.setTags(tags);
+				.setVersion(version);
 		config = builder.build();
 	}
 
 	@AfterClass
 	public static void tearDownClass() {
-		p.destroyForcibly();
```
Comment (cschroed-usgs) [inline src/test/java/gov/usgs/cida/microservices/catalog/ConsulRegistrationAndDiscoveryClientsTestIT.java]: This method is not present in [Java 7](http://docs.oracle.com/javase/7/docs/api/java/lang/Process.html). It is present in [Java 8](http://docs.oracle.com/javase/8/docs/api/java/lang/Process.html#destroyForcibly--) though.

Comment (isuftin) [inline src/test/java/gov/usgs/cida/microservices/catalog/ConsulRegistrationAndDiscoveryClientsTestIT.java]: @cschroed-usgs (!!!!) serves me right for using JDK8 too soon. :/

Comment (cschroed-usgs) [inline src/test/java/gov/usgs/cida/microservices/catalog/ConsulRegistrationAndDiscoveryClientsTestIT.java]: Haha, no worries. Java 8 is certifiably neat-o.

Comment (cschroed-usgs) [inline src/test/java/gov/usgs/cida/microservices/catalog/ConsulRegistrationAndDiscoveryClientsTestIT.java]: (TRADEMORK)

**Pull request merged**